doc/build.md

```diff
@@ -23,22 +23,26 @@ Before building the project, consult the following table to ensure you use the r
 
 #### 2. Download and Build Pixar USD 
 
-See Pixar's official github page for instructions on how to build USD: https://github.com/PixarAnimationStudios/USD
+See Pixar's official github page for instructions on how to build USD: https://github.com/PixarAnimationStudios/USD . Pixar has recently removed support for building Maya USD libraries/plug-ins in their github repository and ```build_usd.py```. When building the maya-usd project, it is important the recommended ```Pixar USD``` and ```Animal Logic``` commitID or tag from the table below are used: 
 
-For additional information on building USD, see the ***Building Pixar USD*** section below.
+|   Required    |      ![](images/pxr.png)          |         ![](images/al.png)            |
+|:------------: |:---------------:  |:-----------------:    |
+| CommitID/tag  | ceca736(>19.11)   | 631a2911(>0.34.0)     |
+
+
+For additional information on building Pixar USD, see the ***Additional Build Instruction*** section below.
```

Comment:
**USD version compatibility**  is now in the same page.  Additional information on building Pixar USD can also be found in the same page under ***Additional Build Instruction*** section.